pkg/server/events.go

```diff
@@ -42,6 +42,13 @@ const (
 	backOffInitDuration        = 1 * time.Second
 	backOffMaxDuration         = 5 * time.Minute
 	backOffExpireCheckDuration = 1 * time.Second
+
+	// handleEventTimeout is the timeout for handling 1 event. Event monitor
+	// handles events in serial, if one event blocks the event monitor, no
+	// other events can be handled.
+	// Add a timeout for each event handling, timeout events will be requeued and
```

Comment:
nit
/s/timeout events/events that timeout/

Reply:
Done